raiden-ts/src/transfers/epics/locked.ts

```diff
@@ -853,7 +850,7 @@ function sendWithdrawRequest(
       );
       if (action.payload?.coopSettle !== undefined) {
         // coopSettle reply has a more relaxed constraint on expiration
-        assert(action.meta.expiration > state.blockNumber, ErrorCodes.CNL_WITHDRAW_EXPIRES_SOON);
+        assert(action.meta.expiration > Date.now() / 1e3, ErrorCodes.CNL_WITHDRAW_EXPIRES_SOON);
```

Comment:
If we're comparing `number`s, we don't need `Math.floor` as JS can compare integers and floats natively. `expiration` is in seconds here as well, that's why we divide `now / 1e3`. Unfortunately, we don't have this consistency here, since this PR is mostly the first time we introduce seconds, and I didn't want to refactor/rename several variables which were just internal and could hold the values without issues besides the consistency. For now, one still needs to refer to the structures declarations, but mostly numeric timeouts coming from contracts and messages are in seconds, and "runtime" timeouts (like `httpTimeout`) are kept in the native millisecond unit.